packages/odyssey-react/src/components/Table/TableSortButton.tsx

```diff
@@ -41,13 +76,20 @@ const TableSortButton = forwardRef<Ref, Props>((props, ref) => {
     <button ref={ref} className={componentClass} {...omitProps}>
       {children}
       <span className={styles.sortIndicator}>
-        {direction === "unsorted" && <SortIcon title="Unsorted" />}
-        {direction === "asc" && <SortAscIcon title="Ascending" />}
-        {direction === "desc" && <SortDescIcon title="Descending" />}
+        {direction === "unsorted" && <SortIcon title={unsortedIconTitle} />}
+        {direction === "asc" && <SortAscIcon title={ascendingIconTitle} />}
+        {direction === "desc" && <SortDescIcon title={descendingIconTitle} />}
```

Comment:
❤️ thanks for the i18n fix - I believe I created a JIRA in our backlog you can close out now!